src/ServiceInsight/Saga/SagaUpdateControl.xaml

```diff
@@ -23,108 +22,38 @@
                 <Setter Property="Foreground" Value="{StaticResource SagaBlue}" />
                 <Setter Property="HorizontalContentAlignment" Value="Left" />
                 <Setter Property="VerticalContentAlignment" Value="Top" />
-                <Setter Property="Padding" Value="4,1,0,0" />
-                <Setter Property="BorderThickness" Value="1" />
-                <Setter Property="BorderBrush">
-                    <Setter.Value>
-                        <LinearGradientBrush StartPoint="0.5,0" EndPoint="0.5,1">
-                            <GradientStop Offset="0" Color="#FFA3AEB9" />
-                            <GradientStop Offset="0.375" Color="#FF8399A9" />
-                            <GradientStop Offset="0.375" Color="#FF718597" />
-                            <GradientStop Offset="1" Color="#FF617584" />
-                        </LinearGradientBrush>
-                    </Setter.Value>
-                </Setter>
+                <Setter Property="Padding" Value="0,0,0,1" />
+                <Setter Property="FontSize" Value="10" />
```

Comment:
Should this explicitly say `10px`? There's some inconsistency in the file, but in most instances `px` is stated explicitly.